src/index.js

```diff
@@ -242,11 +245,20 @@
    * @private
    * @returns {String} Time in H:mm format with am/pm, e.g. '9:30am'
    */
-  function hmmAmPm(date) {
+  function hmmAmPm(date,timeFormat) {
```

Comment:
Sorry, was too busy but flagged this to respond :) Your idea is good. Additionally it would be good to add a 'locale' option if you want to further improve it. I just needed those two options that's why I made it as simple as with timeFormat :)

Thanks!